packages/lib-sourcify/src/lib/hashFunctions/swarmHash.ts

```diff
@@ -0,0 +1,68 @@
+import { makeChunkedFile } from '@fairdatasociety/bmt-js';
```

Comment:
Here is was more a collab between me and ChatGpt 😆 I wrote swarmBzzr1Hash because ChatGpt couldn't create a working version of it while translating from c++. Everything was translated by ChatGPT.